libs/liborcania/Makefile

```diff
@@ -7,13 +7,13 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=liborcania
 PKG_VERSION:=2.3.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/babelouest/orcania/tar.gz/v$(PKG_VERSION)?
 PKG_HASH:=b1b5550523164eca0f59099e843177684c5017c6088284123880cffd4c6dbbde
 
-PKG_MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
+PKG_MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>, Austin Lane <vidplace7@gmail.com>
```

Comment:
This change is somehow unreleated to install library on the target, though.

Reply:
@BKPepe I'm happy to remove this change, just trying to convey that I am happy to maintain this into the future.

Reply:
Removed myself as a maintainer for now.